internal/handlers/staking_restrictions_hooks.go

```diff
@@ -0,0 +1,141 @@
+package handlers
+
+import (
+	"cosmossdk.io/math"
+	"github.com/cosmos/cosmos-sdk/simapp/helpers"
+	sdktypes "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	stakingkeeper "github.com/cosmos/cosmos-sdk/x/staking/keeper"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+)
+
+// Assert proper implementation of StakingRestrictions
+var _ stakingtypes.StakingHooks = StakingRestrictionHooks{}
+
+const (
+	// The default concentration of bonded tokens a validator is allowed as a multiple of equal shares
+	DefaultConcentration = 5.5 // Amounts to ~8% with 68 validators
```

Comment:
nit: to match with the other properties of the same type (i.e: `MaxConcentrationMultiple`).

```suggestion
	DefaultConcentrationMultiple = 5.5 // Amounts to ~8% with 68 validators
```